### PR TITLE
[Misc] change default gc from CMS to PS

### PR DESCRIPTION
**Pull request by sandlerwang**
Summary: The same as title

Test Plan: hotspot/test/gc/TestDefaultGC.sh

Reviewed-by: kuaiwei, JoshuaZhuwj

Issue: https://github.com/alibaba/dragonwell8/issues/561

### CODE REVIEW

**Comment by kuaiwei** [comment]
It's not an aarch64 specific fix, please change the commit message as "change default gc from CMS to PS"

**Review by tanghaoth90 — APPROVED** [review]
LGTM

**Review by kuaiwei — APPROVED** [review]

**Comment by sandlerwang** [comment]
serviceability/sa/jmap-minidump/TestNoMinidumpAtFullGC.java https://github.com/alibaba/dragonwell8/issues/513

**Comment by sandlerwang** [comment]
> serviceability/sa/jmap-minidump/TestNoMinidumpAtFullGC.java #513

测试中挂掉的case是以前的PR引起的

**Review by JoshuaZhuwj — APPROVED** [review]

File: hotspot/src/share/vm/runtime/arguments.cpp [hunk 1]
```diff
@@ -1605,7 +1605,7 @@ void Arguments::select_gc_ergonomically() {
 
 void Arguments::select_gc() {
   if (!gc_selected()) {
-    FLAG_SET_ERGO(bool, UseConcMarkSweepGC, true);
+    select_gc_ergonomically();
```
Comment (sendaoYan) [inline hotspot/src/share/vm/runtime/arguments.cpp:1608]: select_gc_ergonomically  这个代表什么含义呀

**Pull request merged**